### ISSUE TITLE
Aggregation: Attempting an aggregate on a referenced collection

### ISSUE THREAD

**Issue by VikingsFan** [OData/odata.net#463]
copy from https://github.com/OData/WebApi/issues/634

**Comment by koniq** [comment]
This query throws same type of error : 
https://localhost/v5/odata/contacts?$apply=groupby((Contributions/Amount), aggregate( $count as Amount))

and additionally - assuming that Contributions would be defined as `<Property Name="Contributions" Type="Collection(Edm.String)" />` this would also not work but throws different error : 

> $apply/groupby grouping expression 'Contributions ' must evaluate to a property access value.

I understand that it is different example but it also fits to this issue name.

**Comment by KwalityKoder** [comment]
Hi, we are in a position where we need to use a groupby on a property inside a collection, and are getting the same error message reported here, despite the fact that the Odata query is valid in terms of the OData v4 spec. Any idea will this bug be fixed? It's been open for over 3 years now. We are using v7.3.0 of the Microsoft.AspNet.Odata NuGet package.

**Comment by gogulsekar** [comment]
+1

**Comment by habbes** [comment]
Added to backlog: 1848288

**Comment by koniq** [comment]
> Is there an ETA for this feature? Or is there any workaround available? It's a very important feature for us.

This bug has more than six years, I think that coders are afraid of fixing it

**Comment by nathanvj** [comment]
@koniq Yeah. It was added to the backlog this April 4th, so I'm hoping someone might be working on it...

**Comment by baterja** [comment]
I was trying to sum series values nested as an array under the main metadata-like record and came here to see it's not possible :(

**Comment by DavidM29** [comment]
9 years have passed, this will never be possible it seems.

**Comment by ahdung** [comment]
any news?
